app/src/Components/Base/Content.js

```diff
@@ -8,14 +8,51 @@ import DataSource from './../DataSource/DataSource';
 import NotFoundComponent from '../Error/NotFoundComponent'
 import Login from '../Login/Login';
 
+const PrivateRoute = ({ component: Component, ...rest }) => (
```

Comment:
Just because the JavaScript feature is called rest, doesn't mean we should call the variable according to the feature. Just because we create a variable with `var` doesn't mean we call it `variable`.
Just because we fetch a type with `typeof` doesn't mean, we should name the resulting variable `typeOf`.